workflowproj/operator.go

```diff
@@ -41,6 +41,8 @@ const (
 	LabelService = metadata.Domain + "/service"
 	// LabelWorkflow specialized label managed by the controller
 	LabelWorkflow = metadata.Domain + "/workflow-app"
+	// LabelWorkflowNamespace specialized label managed by the controller indicating the namespace of the workflow
+	LabelWorkflowNamespace = metadata.Domain + "/workflow-namespace"
```

Comment:
Can you refresh my memory here. Why do we need this? The `metadata.namespace` already cares for this info.

Reply:
This is true for most cases, but not for a trigger that is created in a different namespace and in such case, we need LabelWorkflowNamespace to let the operator know which namespace the workflow resides, so that it can delete the trigger when the workflow is removed.